internal/pkg/v2/infrastructure/redis/queries.go

```diff
@@ -146,3 +146,12 @@ func objectIdExists(conn redis.Conn, id string) (bool, errors.EdgeX) {
 	}
 	return exists, nil
 }
+
+func getMemberNumber(conn redis.Conn, command string, key string) (uint32, errors.EdgeX) {
```

Comment:
is there any case we might use this function aside from ZCARD?

Reply:
For all V2 APIs we have implemented so far, using ZCARD is sufficient at this moment.  I intentionally make this func as flexible as possible, so if we do need to use `SCARD` in the future, getMemberNumber still could be leveraged.  If we are confident that there is not need to use SCARD, I'm fine to restrict getMemberNumber with ZCARD only.